qa/standalone/ceph-helpers.sh

```diff
@@ -1368,6 +1371,7 @@ function wait_for_clean() {
         # Comparing get_num_active_clean & get_num_pgs is used to determine
         # if the cluster is clean. That's almost an inline of is_clean() to
         # get more performance by avoiding multiple calls of get_num_active_clean.
+	ceph pg stat # for benefit of someone reading the log
```

Comment:
Looks indentation is not correct.
Rather your comment is also correct.
But I would write as:
  "For the benefit of someone reading the logs"